pyoverkiz/client.py

```diff
@@ -203,7 +203,7 @@ async def nexity_login(self) -> str:
             ),
         )
 
-        aws = AWSSRP(
+        aws = WarrantLite(
```

Comment:
That’s the fix?

Reply:
It seems that both packages are pretty outdated, but that we could use the lite variant as well.